hijack/tests/test_admin.py

```diff
@@ -16,17 +18,39 @@ def test_user_admin(self, admin_client):
         assert b"data-hijack-user" in response.content
 
     @pytest.fixture()
-    def no_user_admin(self, settings):
+    def no_user_admin(self):
+        from django.contrib import admin
+
+        custom_user_admin = admin.site._registry.pop(CustomUser)
+        yield
+        admin.site._registry[CustomUser] = custom_user_admin
```

Comment:
That is not same?
```suggestion
        admin.site.unregister(CustomUser)
        yield
        admin.site.register(CustomUser)
```

Reply:
Sadly, no, it took my a while to figure out how to make the test work properly 🤷